pkgs/misc/cups/drivers/kyodialog3/default.nix

```diff
@@ -1,34 +1,34 @@
-{ stdenv, lib, fetchzip, cups
+{ stdenv, lib, fetchzip, cups, autoPatchelfHook
 
   # Can either be "EU" or "Global"; it's unclear what the difference is
-  , region ? "Global"
+  , region ? "Global", qt4
 }:
 
 let
   platform =
     if stdenv.system == "x86_64-linux" then "64bit"
     else if stdenv.system == "i686-linux" then "32bit"
          else throw "Unsupported system: ${stdenv.system}";
-  debPlatform = 
+  debPlatform =
     if platform == "64bit" then "amd64"
     else "i386";
   debRegion = if region == "EU" then "EU." else "";
-
-  # TODO: add Qt4 for kyodialog3 application
-  libPath = lib.makeLibraryPath [ cups ];
 in
 stdenv.mkDerivation rec {
   name = "cups-kyodialog3-${version}";
   version = "8.1601";
 
-  dontPatchELF = true;
   dontStrip = true;
 
   src = fetchzip {
     url = "https://usa.kyoceradocumentsolutions.com/content/dam/kdc/kdag/downloads/technical/executables/drivers/kyoceradocumentsolutions/us/en/Kyocera_Linux_PPD_Ver_${version}.tar.gz";
     sha256 = "11znnlkfssakml7w80gxlz1k59f3nvhph91fkzzadnm9i7a8yjal";
   };
 
+  nativeBuildInputs = [ autoPatchelfHook ];
```

Comment:
Binaries look good to me, but can you check they still work for you?

Reply:
not all of them work, e.g. there's a QT application that doesn't. the cups filter binaries work fine though; I've been printing with this thing :-)